packages/wrangler/src/config/index.ts

```diff
@@ -463,14 +466,16 @@ export function printBindings(
 
 				if (context.local) {
 					const registryDefinition = context.registry?.[service];
+					hasConnectionStatus = true;
+
 					if (
 						registryDefinition &&
 						(!entrypoint ||
 							registryDefinition.entrypointAddresses?.[entrypoint])
 					) {
-						value = `🟢 ` + value;
+						value = value + " " + chalk.green("connected");
```

Comment:
nit: do you think smth like 👇 would better differentiate between the binding and its status?
```suggestion
						value = value + " " + chalk.green("[connected]");
```

Reply:
+1 on this especially in scenarios when terminal colors are turned off (CI, certain restricted SSH connections or older terminals, etc)